packages/terra-worklist-data-grid/src/proptypes/columnHeaderShape.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+import columnHeaderCellShape from './columnHeaderCellShape';
+
+const columnHeaderShape = PropTypes.shape({
```

Comment:
This should really be a column shape.

Reply:
These were initially separated as we expected different properties for a _th_ vs _td_. But because we don’t have that requirement right now, we will refactor & simplify.